src/scss/_lg-fonts.scss

```diff
@@ -2,6 +2,7 @@
     font-family: 'lg';
     src: url('#{$lg-path-fonts}/lg.ttf?io9a6k') format('truetype'),
         url('#{$lg-path-fonts}/lg.woff?io9a6k') format('woff'),
+        url('#{$lg-path-fonts}/lg.woff2?io9a6k') format('woff2'),
```

Comment:
the woff2 format should come first, as the browser will use the first supported format

Reply:
Thanks for your time. Yes, I'll make the changes